### PR TITLE
transactions

### PR DESCRIPTION
**Pull request by isfgjspquzml**
transactions appear to look fine in portal. see user breadcrumbs and crashes but not system breadcrumbs

### CODE REVIEW

**Comment by isfgjspquzml** [comment]
however, one issue is that transactions don't appear to persist across background/foregrounding but i'm not sure whether that's b/c I'm using a trial version (whenever I background and reopen the application it takes 3s to show the trial screen).

**Comment by isfgjspquzml** [comment]
https://github.com/crittercism/crittercism-docs/pull/102

File: DemoApp/Assets/Plugins/Crittercism_Android_Scripts/CrittercismAndroid.cs [hunk 1]
```diff
@@ -169,13 +169,73 @@ static public void SetValue (string key, string value)
...
 				}
 
-				mCrittercismsPlugin.CallStatic ("leaveBreadcrumb", l);
+				mCrittercismsPlugin.CallStatic ("beginTransaction", l);
 		}
 
+		/// <summary>
+		/// Ends a tracked transaction ex. login was successful
+		/// </summary>
+		static public void EndTransaction (string l)
```
Comment (dshirley) [inline DemoApp/Assets/Plugins/Crittercism_Android_Scripts/CrittercismAndroid.cs]: nit: please use a more meaningful variable name like "transactionName"

Comment (isfgjspquzml) [inline DemoApp/Assets/Plugins/Crittercism_Android_Scripts/CrittercismAndroid.cs]: done

**Comment by isfgjspquzml** [comment]
for consistency, I changed the indentation from 8 spaces to 4 spaces in one of the CrittercismAndroid files

**Comment by dshirley** [comment]
LGTM

**Pull request merged**